api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -343,7 +343,7 @@ public FirecloudWorkspace cloneWorkspace(
                 ImmutableList.of(new FirecloudManagedGroupRef().membersGroupName(authDomainName)));
 
     return retryHandler.run(
-        (context) -> workspacesApi.cloneWorkspace(fromProject, fromName, cloneRequest));
+        (context) -> workspacesApi.cloneWorkspace(cloneRequest, fromProject, fromName));
```

Comment:
Swagger 3's order changed, before: Body is the last item, now Body is the first

Reply:
If you haven't already, please check any usages of these APIs to verify there aren't any ordering changes which weren't caught by compilation - for example foo(String x, String y) -> foo(String y, String x)

Reply:
seems this changed `body` vs `parameter(path and query)`. And since Body will be object, so I think `String x, String y` swapping would not happen IMO

Reply:
Is the body param required to be an object?  Or is it possible we have specified a string-body somehow?

Reply:
This part of the mustache file makes me suspicious:
```
{{! Begin AoUPlainTextBodyFix }}
          } else if (obj instanceof String) {
              return RequestBody.create(MediaType.parse(contentType), (String) obj);
          {{! End AoUPlainTextBodyFix }}
```

Reply:
I just did a code search, and does find something I was missing: https://github.com/all-of-us/workbench/pull/4953/commits/582797a2afe2112031dec123c9abdfc66b7e7c4a
Now I think all body should be json object...